<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SQLAlina</class>
 <widget class="QWidget" name="SQLAlina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQLAlina</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>671</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="View">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Просмотр&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorDate_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>671</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="PathB"/>
    </item>
    <item>
     <widget class="QPushButton" name="ConnectDatabase">
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReturnBD">
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="connectError">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Таблица</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="Zapros">
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ZaprosButton">
      <property name="text">
       <string>Запрос  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ConnectandInquiry_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Создать таблицу&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="TableN">
      <property name="text">
       <string>Название таблицы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TCreate">
      <property name="text">
       <string>Создать таблицу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TDrop">
      <property name="text">
       <string>Удалить Таблицу</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>671</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="tableName">
      <property name="text">
       <string>Название таблицы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameColumn">
      <property name="text">
       <string>Название столбца</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="Type">
      <item>
       <property name="text">
        <string>INTEGER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TEXT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DATE</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CreateColumn">
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Error_Create">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLineEdit" name="StringDate">
      <property name="text">
       <string>Строка</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Addstring">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteDate">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
